integrations/dotnet/workload-executor

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
```

Comment:
Is this really required now that things are tested and we know that this script does what it is supposed to? I think not. If that's right, you can also get rid of a lot of the `set +/-x` statements further down in this script.